Library/Formula/llvm.rb

```diff
@@ -118,6 +118,8 @@ def install
       args << "-DCMAKE_OSX_ARCHITECTURES=#{Hardware::CPU.universal_archs.as_cmake_arch_flags}"
     end
 
+    args << "-DCMAKE_CXX_FLAGS=#{ENV['CPPFLAGS']}" unless OS.mac?
```

Comment:
Can you add a comment indicating which error message this line is intended to fix?

Should this include both `CPPFLAGS` and `CXXFLAGS`?
```ruby 
args << "-DCMAKE_CXX_FLAGS=#{ENV['CPPFLAGS'] + ENV['CXXFLAGS']}" unless OS.mac?
```